fabric-rendering-fluids-v1/src/main/java/net/fabricmc/fabric/api/client/render/fluid/v1/SimpleFluidRenderHandler.java

```diff
@@ -62,9 +65,7 @@
 	 */
 	public static final Identifier LAVA_FLOWING = new Identifier("block/lava_flow");
 
-	protected final Identifier stillTexture;
-	protected final Identifier flowingTexture;
-	protected final Identifier overlayTexture;
```

Comment:
Would this class ever be extended? As these fields are technically exposed by the api to mods that extend this?

Reply:
Fixed